contrib/automation_tests/orbit_capture_loading.py

```diff
@@ -0,0 +1,81 @@
+"""
```

Comment:
You might want to add the flag file right away, no point in delaying it to a second PR I think.

Reply:
Done.